client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
I kind of agree with @aduth that this is part of UI and probably for now lets leave it in `ui`
And I dont think we should notices as loggers for other actions, I would like to see it the other way around - notices beeing a by-product of other actions